lib/cucumber/formatter/console.rb

```diff
@@ -85,8 +85,8 @@ def print_stats(features, options)
         @io.puts scenario_summary(runtime) {|status_count, status| format_string(status_count, status)}
         @io.puts step_summary(runtime) {|status_count, status| format_string(status_count, status)}
 
-        @io.puts(format_duration(features.duration)) if features && features.duration
-
+        @io.puts(format_duration(features.duration)) if features && features.duration && options[:duration]
```

Comment:
I wonder if we still need that `features.duration` check. Could it ever be nil? Looks quite defensive.